deepchecks/tabular/metric_utils/scorers.py

```diff
@@ -293,7 +293,18 @@ def _run_score(self, model, dataset: 'tabular.Dataset'):
                 f'{doclink_str}')
         if self.possible_classes is not None:
```

Comment:
add comment that this case is classification

Reply:
Don't classes imply classification?